src/osemgrep/core/Metrics_.ml

```diff
@@ -80,18 +82,23 @@ open Semgrep_metrics_t
 (* Types and constants *)
 (*****************************************************************************)
 
-(* TODO: set to the cannonical "metrics.semgrep.dev" once we upgrade the
- * host from TLS 1.2 to TLS 1.3
+(* TODO: set to the cannonical "https://metrics.semgrep.dev" once we upgrade
+ * the host from TLS 1.2 to TLS 1.3
  *)
 let metrics_url =
   Uri.of_string "https://oeyc6oyp4f.execute-api.us-west-2.amazonaws.com/Prod/"
 
 (*
      Configures metrics upload.
 
-     ON - Metrics always sent
-     OFF - Metrics never sent
-     AUTO - Metrics only sent if config is pulled from the server
+     On   - Metrics always sent
+     Off  - Metrics never sent
+     Auto - Metrics only sent if config is pulled from the registry
+            or if using the Semgrep App.
+
+  What is the rational for Auto? I guess if the user is requesting rules
```

Comment:
I'm also for removing auto and making `metrics-off` / `disable-metrics` a flag

Reply:
yes, might be simpler. Not sure entirely what was the rational for Auto to be honest. Maybe you'll need to talk to luke or the people who wrote PRIVACY.md (maybe isaac).